packages/splits-sdk/src/client/splitV1.ts

```diff
@@ -670,15 +664,28 @@ export class SplitV1Client extends SplitV1Transactions {
     })
     const event = events.length > 0 ? events[0] : undefined
     if (event) {
-      const log = decodeEventLog({
-        abi: this._splitMainAbi,
```

Comment:
was this not working properly? setting `_splitMainAbi` based on `_chainId`?

Reply:
you know, another minor thing we should do. at the beginning of the function, we should set a local variable for `this._chainId` (or `this._splitMainAbi` if we're gonna keep that).

Cause technically, it could change mid execution. i.e. you call create split, and while it's still running you could change `this._chainId` to a new network. So we should trap it in a local variable at the beginning of the function call.

Reply:
maybe let's do that in a separate pr. technically need to do it everywhere 😢

Reply:
yeah typescript was throwing errors, I think the flow would still go through but eslint forced me to change this